scripts/ci/pre_cache.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -u
+
+# if there is no directory for this $CI_COMMIT_REF_NAME/$CI_JOB_NAME
+# create such directory and
+# copy recursively all the files from the newest dir which has $CI_JOB_NAME, if it exists
+
+# cache lives in /ci-cache/${CI_PROJECT_NAME}/${2}/${CI_COMMIT_REF_NAME}/${CI_JOB_NAME}
+
+function prepopulate {
+  if [[ ! -d $1 ]]; then
+    mkdir -p "/ci-cache/$CI_PROJECT_NAME/$2/$CI_COMMIT_REF_NAME";
+    FRESH_CACHE=$(find "/ci-cache/$CI_PROJECT_NAME/$2" -mindepth 2 -maxdepth 2 \
+      -type d -name "$CI_JOB_NAME"  -exec stat --printf="%Y\t%n\n" {} \; |sort -n -r |head -1 |cut -f2);
+    if [[ -d $FRESH_CACHE ]]; then
+      echo "____Using" "$FRESH_CACHE" "to prepopulate the cache____";
+      time cp -r "$FRESH_CACHE" "$1";
+    else
+      echo "_____No such $2 dir, proceeding from scratch_____";
+    fi
+  else
+    echo "____No need to prepopulate $2 cache____";
+  fi
+}
+
+# CARGO_HOME cache is still broken so would be handled some other way.
+prepopulate "$CARGO_TARGET_DIR" targets
```

Comment:
What does `prepopulate` command do? Or is it a typo?

Reply:
https://github.com/paritytech/cumulus/pull/527/files#diff-ec994b132c375339c6af70baf02771519b29ebfb102e2a49c62059652edf8517R11

Reply:
Oh, sorry, dunno how I've missed that 😅